packages/schematics/angular/application/index_spec.ts

```diff
@@ -44,23 +44,23 @@ describe('Application Schematic', () => {
 
     const tree = schematicRunner.runSchematic('application', options, workspaceTree);
     const files = tree.files;
-    expect(files.indexOf('/projects/foo/karma.conf.js')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/tsconfig.app.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/tsconfig.spec.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/tslint.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/environments/environment.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/environments/environment.prod.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/favicon.ico')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/index.html')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/main.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/polyfills.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/styles.css')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/test.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/app/app.module.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/app/app.component.css')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/app/app.component.html')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/app/app.component.spec.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/projects/foo/src/app/app.component.ts')).toBeGreaterThanOrEqual(0);
+    expect(files).toContain('/projects/foo/karma.conf.js');
```

Comment:
Even better you could use (for long lists like that) https://jasmine.github.io/2.2/introduction#section-Partial_Array_Matching_with_%3Ccode%3Ejasmine.arrayContaining%3C/code%3E 😄

Reply:
Oh sweet :), Ill try to fit that change in sometime 👍

Reply:
Oh sweet :), Though with `not` it might not work as good

example

```
 expect(['a', 'c']).toEqual(jasmine.arrayContaining(['a', 'b')]) -> false
```